src/components/Button/Button.scss

```diff
@@ -643,3 +661,31 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
 .DisclosureIconFacingUp {
   transform: rotate(-180deg);
 }
+
+.ConnectedDisclosureWrapper {
+  display: flex;
+  align-items: center;
+}
+
+.ConnectedDisclosure {
+  z-index: z-index(segment, $stacking-order);
+  margin-left: -(border-width());
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+
+  // Because the outline border color has a 40% opacity, the left border appears darker than the rest of the borders because they are layered over one another. Reducing the opacity to zero for the connected disclosure when not focused gives us the expected border color.
+  &.outline:not(:focus) {
+    border-left-color: rgba(color('ink', 'lighter'), 0);
```

Comment:
Keep in mind, functions like `rgba` and `darken` won't work under the new color system with CSS Custom Properties, since theming happens client side, and these functions are all run at build time.